cmd/crates/soroban-spec-typescript/fixtures/test_custom_types/dist/cjs/constants.d.ts

```diff
@@ -0,0 +1,18 @@
+/**
+ * The Soroban contract ID for the test_custom_types contract.
+ */
+export declare const CONTRACT_ID = "CA3D5KRYM6CB7OWQ6TWYRR3Z4T7GNZLKERYNZGGA5SOAOPIFY6YQGAXE";
```

Comment:
should the `dist/` folders be committed?

Reply:
I generally prefer to have the minimum and have it being dynamically built.. but if it's not causing a concrete issue, we can drop these later on. ( note that it would remain in the github history forever - but I don't consider that to be an issue ).